Source/FolioReaderPage.swift

```diff
@@ -78,21 +78,26 @@ class FolioReaderPage: UICollectionViewCell, UIWebViewDelegate, UIGestureRecogni
     }
     
     func webViewFrame() -> CGRect {
-        let paddingTop: CGFloat = 20
+
+		if (readerConfig.hideBars == true) {
```

Comment:
You can use `guard` here is more Swifty

Need to test, but something like this:
```swift
guard readerConfig.hideBars == false else { return UIScreen.mainScreen().bounds }
```